myTeam.py

```diff
@@ -65,6 +65,19 @@ def bfs_until_non_wall(start, game_state):
     :param start: Tuple (x, y) representing the start coordinate.
     :return: List of tuples representing the path to the first non-wall position.
     """
+
+    # Correct start if its coordinates are negative
+    if start[0] < 0:
+        start = (0, start[1])
+    if start[1] < 0:
+        start = (start[0], 0)
+
+    # Correct start if its coordinates are greater than the width or height of the board
+    if start[0] > game_state.data.layout.width:
+        start = (game_state.data.layout.width, start[1])
+    if start[1] > game_state.data.layout.height:
+        start = (start[0], game_state.data.layout.height)
```

Comment:
The grid starts counting at 0, so I think this should check for `>=` and set to `game_state.data.layout.width/height - 1`
```suggestion
    if start[0] >= game_state.data.layout.width:
        start = (game_state.data.layout.width - 1, start[1])
    if start[1] >= game_state.data.layout.height:
        start = (start[0], game_state.data.layout.height - 1)
```
But do we really not trust the game that much that we need to do this check? 😂